lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -31,15 +31,19 @@ using namespace glow;
 //===----------------------------------------------------------------------===//
 
 // This is the floating point implementation of Convolution.
-void InterpreterFunction::fwdConvolutionInst_FloatImpl(
+template <typename ElemTy>
+typename std::enable_if<
```

Comment:
Why do we need the enable_if section? If we need to protect the usage of this method, could we insert a static_assert into the body of the function to keep the declaration simple?

Reply:
Good point.
Done!